ethportal-peertest/src/scenarios.rs

```diff
@@ -0,0 +1,42 @@
+use crate::jsonrpc::{
+    make_ipc_request, validate_portal_offer, JsonRpcRequest, HISTORY_CONTENT_KEY,
+    HISTORY_CONTENT_VALUE,
+};
+use crate::{Peertest, PeertestConfig};
+use log::info;
+use serde_json::{json, Value};
+use trin_core::jsonrpc::types::Params;
+
+pub fn test_offer_accept(peertest_config: PeertestConfig, peertest: &Peertest) {
+    info!("Testing OFFER/ACCEPT flow");
+
+    // Store content to offer in the testnode db
+    let store_request = JsonRpcRequest {
+        method: "portal_historyStore".to_string(),
+        id: 11,
+        params: Params::Array(vec![
+            Value::String(HISTORY_CONTENT_KEY.to_string()),
+            Value::String(HISTORY_CONTENT_VALUE.to_string()),
+        ]),
+    };
+
+    let store_result = make_ipc_request(&peertest_config.target_ipc_path, &store_request).unwrap();
+    assert_eq!(store_result.as_str().unwrap(), "true");
+
+    // Send offer request
```

Comment:
A bit more of a descriptive comment could be nice ... `// Send offer request from testnode to bootnode`?